src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -13,6 +14,7 @@
 import seedu.addressbook.data.person.UniquePersonList;
 import seedu.addressbook.data.tag.Tag;
 import seedu.addressbook.data.tag.UniqueTagList;
+import seedu.addressbook.ui.TextUi;
 
 /**
  * Adds a person to the address book.
```

Comment:
Remember to update the header comment of the class to reflect the latest changes.